proxy/ui.coffee

```diff
@@ -69,6 +70,18 @@ class WebExtUI extends UI
       @postActive()
     ), 60 * 60 * 1000
 
+  initToggle: ->
+    try
+      getting = chrome.storage.local.get("snowflake-enabled", (result) =>
```

Comment:
What follows here is pretty similar to the body of the `onMessage` handler.  It seems like a common method can be used for both.